api/src/main/java/org/apache/gravitino/authorization/SecurableObjects.java

```diff
@@ -120,6 +136,22 @@ public static SecurableObject ofFileset(
     return of(MetadataObject.Type.FILESET, names, privileges);
   }
 
+  /**
```

Comment:
We don't support the privilege of the model now. You can remove this temporaly.

Reply:
ok, removed.